nearcore/src/runtime/mod.rs

```diff
@@ -629,6 +629,10 @@ impl RuntimeAdapter for NightshadeRuntime {
         (self.store.clone(), self.genesis_state_roots.clone())
     }
 
+    fn get_store(&self) -> Arc<Store> {
+        self.store.clone()
+    }
```

Comment:
This is a bit weird. I think it is intentional that we do not want to expose `Store` from `NightshadeRuntime` to avoid breaking the abstraction.

Reply:
But we already expose store through genesis_state method (which returns (store, state_roots))

Reply:
We should probably fix that instead of taking it for granted

Reply:
The method we use from runtime would not make sense if runtime do not have access to state